components/service/glean/docs/baseline.md

```diff
@@ -0,0 +1,18 @@
+# The `baseline` ping
+This ping is intended to provide metrics that are managed by the library itself, and not explicitly
+set from the application or included in the application's `metrics.yaml` file.
```

Comment:
```suggestion
set by the application or included in the application's `metrics.yaml` file.
```

Reply:
Thanks Chris, this is very helpful! I'm making the changes locally and then I'll force-push here (vs accepting the changes here online).